Linux Headers missing from Dockerfile

<!--
If this is a bug report please fill out the template below.

If this is a feature request please describe the behavior that you'd like to see.
-->

**Environment**:

  - CTFd Version/Commit: latest ( file in question is https://github.com/CTFd/CTFd/blob/master/Dockerfile found in commit https://github.com/CTFd/CTFd/commit/c8031b38c27926b8166174af331bc2b4ab614c1b )
  - Operating System: Ubuntu 18.04
  - Web Browser and Version: N/A

**What happened?**
When running `docker compose build` with a plugin that has additional python pip packages that require linux headers to install, pip fails to install the additional package.

**What did you expect to happen?**
I expected the plugin to install correctly. 

**How to reproduce your issue**
Add this package to a requirements.txt, either in the plugin or in the main requirements:
`python-openstackclient`
(see: https://git.cybbh.space/ben/ctfd_openstack_integration/blob/master/requirements.txt ) 

**Any associated stack traces or error logs**
netifaces.c:131:28: fatal error: linux/tipc.h: No such file or directory pip


**Proposed solution**
Modify Dockerfile from:
`apk add python python-dev libffi-dev gcc make musl-dev py-pip mysql-client git openssl-dev`
to
`apk add python python-dev libffi-dev gcc make musl-dev py-pip mysql-client git openssl-dev linux-headers`

I can fork and make a PR if necessary.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,6 +1,6 @@
 FROM python:2.7-alpine
 RUN apk update && \
-    apk add python python-dev libffi-dev gcc make musl-dev py-pip mysql-client git openssl-dev
+    apk add python python-dev linux-headers libffi-dev gcc make musl-dev py-pip mysql-client git openssl-dev
 
 WORKDIR /opt/CTFd
 RUN mkdir -p /opt/CTFd

